attending/library.py

```diff
@@ -159,6 +160,13 @@ def fetch_via_name(module, version=None, url=None):
                 raise options_exhausted(module, version)
         else:
             return fetch_via_module(python_module, version)
-    
     return lib.fetch(module, version, url)
 
+
+def fetch_via_local_index(module, home=Path().home()):
```

Comment:
Let's keep the interface of the other fetch modules
```suggestion
def fetch_via_local_index(module, version=None, home=Path().home()):
```
and specify a placeholder for where the version portion of the string goes in the URL. That way we don't hardcode the urls in the CSV to a particular version.